src/sagas/handler/User.jsx

```diff
@@ -0,0 +1,21 @@
+import { call, put } from 'redux-saga/effects';
+
+import { requestGetUser } from '../requests/User';
+import { setUser } from '../../auth/User';
+
+export function* handleGetUser(action) {
+    try {
+        if (!action.payload) {
```

Comment:
You don't really need to explicitly check if payload is defined. if a null/undefined request body is sent to the API, it should/will respond with an error. Which will in turn be caught in this function (by the catch block)